package.json

```diff
@@ -11,7 +11,7 @@
     "build": "turbo run build --filter=astro --filter=create-astro --filter=\"@astrojs/*\" --filter=\"@benchmark/*\"",
     "build:ci": "turbo run build:ci --filter=astro --filter=create-astro --filter=\"@astrojs/*\" --filter=\"@benchmark/*\"",
     "build:examples": "turbo run build --filter=\"@example/*\"",
-    "dev": "turbo run dev --parallel --filter=astro --filter=create-astro --filter=\"@astrojs/*\" --filter=\"@benchmark/*\"",
+    "dev": "turbo run dev --concurrency=40 --parallel --filter=astro --filter=create-astro --filter=\"@astrojs/*\" --filter=\"@benchmark/*\"",
```

Comment:
I don't think we need the `--concurreny` here. The [docs](https://turbo.build/repo/docs/reference/command-line-reference/run#--concurrency) mentioned that:

> This option is ignored if the [--parallel](https://turbo.build/repo/docs/reference/command-line-reference/run#--parallel) flag is also passed.

Presumably the `--parallel` flag also made the `"persistent": true` config not needed, but I think it's good to have the latter anyways.

Reply:
Hm, I actually hit a build error stating the default `--concurrency` is `10`, and we needed to bump to (# tasks) to get it working. We have 33 CLI tasks which seems to hit the parallel default limit. I chose 40 to give us a bit of room 🤷

Reply:
Strange. Must be outdated docs then. Let's go with this for now!